src/components/board/TagsTabSidebard.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+	<div>
+		<ul class="labels">
+			<li v-for="label in labels" :key="label.id">
+				<template v-if="editingLabelId === label.id">
+					<input v-model="editingLabel.title">
+					<compact-picker :value=editingLabel.color @input="updateColor" :palette=defaultColors></compact-picker>
+					<button class="icon-checkmark" @click="updateLabel(label)" :disabled="!editLabelObjValidated" 
+					v-tooltip="{content: missingDataLabel, show: !editLabelObjValidated, trigger: 'manual' }" />
+
+					<button v-tooltip="t('deck', 'Cancel')" class="icon-close" @click="editingLabelId = null" />
+				</template>
+				<template v-else>
+					<span :style="{ backgroundColor: `#${label.color}`, color:textColor(label.color) }" class="label-title">
+						<span v-if="label.title">{{ label.title }}</span><i v-if="!label.title"><br></i>
```

Comment:
Please just use `<span>{{ label.title }}</span>` and make sure the size of empty entries is properly set via CSS instead of the `<i><br /></i>`